applications/asset_tracker/src/main.c

```diff
@@ -1631,6 +1672,13 @@ void cloud_event_handler(const struct cloud_backend *const backend,
 		if (err) {
 			LOG_ERR("Error processing PGPS packet: %d", err);
 		}
+#endif
+#if defined(CONFIG_NRF_CLOUD_CELL_POS)
```

Comment:
This should probably be moved above the AGPS processing... if both are enabled it looks like it will break before getting this far.

Reply:
Funny, I was just starting to try with both AGPS and CELL_POS enabled.  Yeah, better to move it.